packages/create-astro/src/index.ts

```diff
@@ -227,23 +225,31 @@ export async function main() {
 		);
 	}
 
-	console.log('\nNext steps:');
-	let i = 1;
+	const gitResponse = await prompts({
+		type: 'confirm',
+		name: 'git',
+		message: 'Initialize an empty git repository?',
+		initial: true,
+	});
+
+	if (!gitResponse) {
+		process.exit(0);
+	}
+
+	if (gitResponse.git && !args.dryrun) {
+		await execaCommand('git init', { cwd });
+	}
+
+	console.log(`\n${bgCyan(black(' Next steps '))}\n`);
 	const relative = path.relative(process.cwd(), cwd);
 	if (relative !== '') {
-		console.log(`  ${i++}: ${bold(cyan(`cd ${relative}`))}`);
+		console.log(`cd ${relative}`);
 	}
-
 	if (!installResponse.install) {
-		console.log(`  ${i++}: ${bold(cyan(`${pkgManager} install`))}`);
+		console.log(`${pkgManager} install`);
 	}
-	console.log(
-		`  ${i++}: ${bold(
-			cyan('git init && git add -A && git commit -m "Initial commit"')
-		)} (optional step)`
-	);
 	const runCommand = pkgManager === 'npm' ? 'npm run dev' : `${pkgManager} dev`;
-	console.log(`  ${i++}: ${bold(cyan(runCommand))}`);
+	console.log(runCommand);
```

Comment:
Thoughts on joining all of these next steps into a single command for ultimate copy+paste support?

`cd ./my-astro-project && npm run dev`

Reply:
If so, I think it'd be nice to keep the colored output to highlight each command while keeping the `&&` white or dim.

Reply:
@natemoo-re I heard some feedback from @JuanM04 that `&&` could pose issues on Windows. Wasn't sure myself, so I thought keeping a newline would be safest 🤷

Reply:
The thing with running `some command && another command` is that you aren't running a program per-se, you are now interacting with the shell itself (which might work different in other OSs). [Here is some documentation from execa](https://github.com/sindresorhus/execa#execacommandcommand-options).